### PR TITLE
BNH-180: Fixes surrounding availability datepicker

### PR DESCRIPTION
**Pull request by SleepyUser**
-Datepicker now only shows date and not time
-Datepicker now correctly shows when AvailableSoon is selected
-User is only required to enter a date on step six if AvailableSoon is selected
-User may only choose a future date client side.
-Server side, both validation rules above are enforced
-Fixed an issue with server side invalid model redirects for steps 3-5 always redirecting to step 2

### CODE REVIEW

**Comment by emilypatsko** [comment]
I think this looks good, could you add some screenshots/similar to show how it actually looks when 'available soon' is/isn't selected?

**Review by emilypatsko — COMMENTED** [review]
I think this looks good, could you add some screenshots/similar to show how it actually looks when 'available soon' is/isn't selected?

File: web/wwwroot/js/site.js [hunk 1]
```diff
@@ -17,8 +17,17 @@ function checkIfSelectedValueIsTarget(target, id){
     }
 }
 
+
```
Comment (oskarwilliams378) [inline web/wwwroot/js/site.js:20]: extra new line

**Review by oskarwilliams378 — COMMENTED** [review]
One minor comment

**Comment by SleepyUser** [comment]
Any other option selected:
image
Available soon selected:
image
(Because I'm happy it works, bonus screenshot of previous dates being greyed out)
image

**Review by oskarwilliams378 — APPROVED** [review]

**Pull request merged**